internal/handlers/common/aggregations/expression.go

```diff
@@ -12,31 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package types
+package aggregations
 
 import (
 	"strings"
 
+	"github.com/FerretDB/FerretDB/internal/types"
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
-//go:generate ../../bin/stringer -linecomment -type ExpressionErrorCode
+//go:generate ../../../../bin/stringer -linecomment -type ExpressionErrorCode
 
-// ExpressionErrorCode represents FieldPath error code.
+// ExpressionErrorCode represents Expression error code.
 type ExpressionErrorCode int
 
 const (
 	_ ExpressionErrorCode = iota
 
-	// ErrNotFieldPath indicates that field is not a path.
-	ErrNotFieldPath
+	// ErrNotExpression indicates that field is not an expression.
+	ErrNotExpression
 
-	// ErrEmptyFieldPath indicates that path is empty.
-	ErrEmptyFieldPath
+	// ErrInvalidExpression indicates that expression is invalid.
+	ErrInvalidExpression
 
-	// ErrInvalidFieldPath indicates that path is invalid.
-	ErrInvalidFieldPath
+	// ErrEmptyFieldPath indicates that field path expression is empty.
+	ErrEmptyFieldPath
```

Comment:
```suggestion
	ErrEmptyExpression
```
To be more consistent with other namings, it looks like `$`.

Reply:
I thought that using FieldPath here makes sense as the error is related to specific type of Expression and not all Expressions in general.

Reply:
yeah that's sensible.